CONTRIBUTING.md

```diff
@@ -124,7 +124,7 @@ provide as much detail and context as possible.
 
 ## Pull requests
 
-Good pull requests—patches, improvements, new features—are a fantastic
+Good pull requests - patches, improvements, and new features - are a fantastic
```

Comment:
Well, the reason I made this change is because when I first read this line without the spaces, it seems like "request-patches" was one hyphenated word. The spaces symbolize that it is rather  offset parenthetical information. (See: http://www.grammarly.com/handbook/punctuation/dash/2/dash-parenthetical-information/)